models/fivetran_log__connector_status.sql

```diff
@@ -1,10 +1,17 @@
-with connector_log as (
+with transformation_removal as (
 
-    select *,
-        sum( case when event_subtype in ('sync_start') then 1 else 0 end) over ( partition by connector_id 
-            order by created_at rows unbounded preceding) as sync_batch_id
+    select *
     from {{ ref('stg_fivetran_log__log') }}
+    where event_subtype != 'TRANSFORMATION'
```

Comment:
alternatively you could just filter to the event types/subtypes from the next CTE like
`where event_type in ('SEVERE', 'WARNING') or event_subtype like 'sync%' or event_subtype = 'status'`

Reply:
> i think it should be more like `where event_subtype not like 'transformation%'`
> 
> pretty sure subtypes are all lowercase and the transformation-related event subtypes are `transformation_run_start`, `transformation_run_succeeded `, and `transformation_run_failed` (see https://fivetran.com/docs/logs#events in the basic transformations section)

That's a good point. Maybe it would be easier to just to `where transformation_id is null` thoughts?

Reply:
> alternatively you could just filter to the event types/subtypes from the next CTE like `where event_type in ('SEVERE', 'WARNING') or event_subtype like 'sync%' or event_subtype = 'status'`

Because we do the JSON parsing in the where clause in the next CTE I found that we needed to isolate this clause to its own CTE. Or am I misunderstanding?

Reply:
oh yeah definitely that! forgot that column existed

Reply:
and looking at our own data it looks like we can rely on that column

Reply:
Just update the model to use `transformation_id`. Let me know if you have any other questions!

Re-requested review 😄